src/main/java/net/jextra/fauxjo/Table.java

```diff
@@ -150,7 +150,13 @@ public String buildBasicSelectStatement( String clause )
             trimmedClause = clause;
         }
 
-        return String.format( "select * from %s %s", fullTableName, trimmedClause );
+        String fields = "*";
+        try {
```

Comment:
Code style for Fauxjo is braces on the following line. Also, please put a comment in the catch clause we are swallowing the exception on purpose.